app/frontend/helpers/connect.ts

```diff
@@ -48,7 +48,7 @@ const connect: Connect = _connect
 
 const useStore: any = _useStore
 
-type UseSelector = <T>(selector: (state: State) => T) => T
+type UseSelector = <T>(selector: (state: State) => T, fn?: (a: Object, b: Object) => boolean) => T
```

Comment:
```suggestion
type UseSelector = <T>(selector: (state: State) => T, compareFn?: (a: Object, b: Object) => boolean) => T
```
I'd name it "compareFn" to make the purpose of the function clear at first sight